CLAUDE.md

```
@@ -1,350 +1,360 @@
+mkdir -p build && cd build
+cmake .. -GNinja -DDUSK_UNITY_BUILD=ON
+ninja -j$(nproc) MultiQ_VST3
+```
+
+Build options:
+- `DUSK_UNITY_BUILD=OFF` (default) — enable with `-DDUSK_UNITY_BUILD=ON` for fewer translation units (Linux only; macOS ObjC++ modules are incompatible)
+- Or use `./rebuild_all.sh --fast` which enables Ninja automatically
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)